cli/test/e2e/package_install_test.go

```diff
@@ -137,6 +137,22 @@ key2: value2
 		}}
 		require.Exactly(t, expectedOutputRows, output.Tables[0].Rows)
 	})
+	
+	logger.Section("package installed list with one package installed with wide option", func() {
```

Comment:
Should we also add a failing package install to show how the error message looks (I think it would just be the standard "see useful error message...")?

Reply:
checking